modules/base/rendering/renderablemodel.h

```diff
@@ -102,13 +117,24 @@ class RenderableModel : public Renderable {
 
     properties::BoolProperty _enableDepthTest;
     properties::OptionProperty _blendingFuncOption;
+    properties::BoolProperty _renderWireframe;
+    properties::FloatProperty _frustumSize;
+
+    properties::BoolProperty _useCache;
+    properties::BoolProperty _castShadow;
+    std::string _lightSource;
+    std::string _shadowGroup;
+    properties::BoolProperty _useOverrideColor;
+    properties::Vec4Property _overrideColor;
```

Comment:
The transparency handling code might have to be updated to take a potential transparent override color into account.